presto-main/src/test/java/com/facebook/presto/eventlistener/TestEventListenerManager.java

```diff
@@ -0,0 +1,547 @@
+                Optional.of(true),
+                "dummySerializedCommitOutput");
+        QueryIOMetadata queryIOMetadata = new QueryIOMetadata(inputs, Optional.of(outputMetadata));
+        return queryIOMetadata;
+    }
+    private static QueryInputMetadata getQueryInputMetadata()
+    {
+        String catalogName = "dummyCatalog";
+        String schema = "dummySchema";
+        String table = "dummyTable";
+        String serializedCommitOutput = "commitOutputDummy";
+        List<String> columns = new ArrayList<>(Arrays.asList("column1", "column2", "column3"));
+        Optional<Object> connectorInfo = Optional.of(new Object());
+        QueryInputMetadata queryInputMetadata = new QueryInputMetadata(
+                catalogName,
+                schema,
+                table,
+                columns,
+                connectorInfo,
+                Optional.empty(),
+                serializedCommitOutput);
+        return queryInputMetadata;
+    }
+
+    private static SplitCompletedEvent createDummySplitCompletedEvent()
+    {
+        Instant now = Instant.now();
+        Instant startTimeDummy = now.minusSeconds(100);
+        Instant endTimeDummy = now.minusSeconds(50);
+        SplitStatistics stats = createDummySplitStatistics();
+        SplitFailureInfo failureInfo = new SplitFailureInfo("Error", "Dummy failure message");
+        SplitCompletedEvent event = new SplitCompletedEvent(
+                "query123",
+                "stage456",
+                "stageExec789",
+                "task012",
+                now,
+                Optional.of(startTimeDummy),
+                Optional.of(endTimeDummy),
+                stats,
+                Optional.of(failureInfo),
+                "dummyPayload");
+        return event;
+    }
+    private static SplitStatistics createDummySplitStatistics()
+    {
+        Duration cpuTime = Duration.ofSeconds(500);
+        Duration wallTime = Duration.ofSeconds(1000);
+        Duration queuedTime = Duration.ofSeconds(120);
+        Duration completedReadTime = Duration.ofSeconds(800);
+
+        long completedPositions = 1500;
+        long completedDataSizeBytes = 10000000L;
+
+        Optional<Duration> timeToFirstByte = Optional.of(Duration.ofSeconds(10));
+        Optional<Duration> timeToLastByte = Optional.empty();
+
+        SplitStatistics splitStatistics = new SplitStatistics(
+                cpuTime,
+                wallTime,
+                queuedTime,
+                completedReadTime,
+                completedPositions,
+                completedDataSizeBytes,
+                timeToFirstByte,
+                timeToLastByte);
+        return splitStatistics;
+    }
+
+    private static void tryDeleteFile(Path path)
+    {
+        try {
+            File file = new File(path.toUri());
+            if (file.exists()) {
+                Files.delete(file.toPath());
+            }
+        }
+        catch (IOException e) {
+            // ignore
```

Comment:
Maybe add a log message:
```suggestion
            log.error(e, "Could not delete file found at [%s]", path);
```

Reply:
Done